polycommit/src/marlin_pc/data_structures.rs

```diff
@@ -234,6 +241,19 @@ pub struct Commitment<E: PairingEngine> {
 }
 impl_bytes!(Commitment);
 
+impl<E: PairingEngine> ToMinimalBitRepresentation for Commitment<E> {
+    fn to_minimal_bit_representation(&self) -> Vec<bool> {
+        let comm_bits = self.comm.to_minimal_bit_representation();
+        let shifted_comm_bits = if let Some(shifted_comm) = &self.shifted_comm {
+            shifted_comm.to_minimal_bit_representation()
+        } else {
+            vec![]
+        };
+
+        [comm_bits, shifted_comm_bits].concat()
```

Comment:
```suggestion
        if let Some(shifted_comm) = &self.shifted_comm {
            [comm_bits, shifted_comm.to_minimal_bit_representation()].concat()
        } else {
            vec![]
        }
```
this will avoid a reallocation in the `self.shifted_comm.is_none()` case

Reply:
Edited. Note that the else branch would take `comm_bits` instead of `vec![]`.